### DISCUSSION TITLE
老師禁不完！除ChatGPT還有這些AI槍手

### DISCUSSION THREAD

**Discussion by Xuewen628** [1101-GS3073-C/Python-Community-GS3073-C#585] [General]
下面是我在新聞看到的
https://www.gvm.com.tw/article/98369
近年來AI的發展看似風光無限，但其實其中有很多問題還在等待法規規範
我記得在用於機器學習的資料也存在隱私權問題

**Comment by Raywei0214** [comment]
現在社會競爭激烈，不這樣作弊很難出頭天

**Comment by brian431** [comment]
真的是太愛這些作業小幫手了。看來感覺之後考試比重要佔大一點了，作業大家不是用ai就是互相抄來抄去。

**Comment by magicmicheal** [comment]
AI真的越來越厲害了，以前只有有標準答案的問題，我們才會尋求電腦的幫助，沒想到現在連需要主觀回答的問題AI也能給予幫助，甚至做得比我還要更好，有人可能會開始思考，AI會不會危害到人類，但我只能說心得不用自己想是真的香！

**Comment by awehao** [comment]
看來是AI大戰爭喽，AI的發明雖然給人類帶來了方便，但這也間接造成了很多人的偷吃步，業更兼街的危害了整個社會的運行，這是我們值得深思的點，希望人人都能做到無愧於自己的人生，當個好寶寶，不過心得不用自己想是真的香！，雖然我還是不會用這種方法拉，雖然用AI可能比我做的還好，但算了，我開心就好

**Comment by bww0401** [comment]
訓練模型所蒐集的資料，一直都存在著隱私權的爭議性，所以我認為當科技在進步的同時，法規也要跟進，才能保障這些ai產品能夠在安全的環境下開發出來，以增加人類福祉

**Comment by manipulateed** [comment]
這些AI工具固然能幫助學生學習，但考試就是咬展現自身學習的成果，而不是直接找答案，最重要就是自我的思維模式訓練，才能展現人的原創性與價值。

**Comment by awkward-willy** [comment]
隨著解公式、背誦相關的問題被各種不同的科技破解，未來考試應該會越來越偏向素養考題，著重於學生的看法或解題觀念等，如此也能讓試卷更有鑑別度，並合理展現學生的學習成效。

**Comment by larry40305** [comment]
身為一個有使用photomath來寫作業的人，我認為這類產品對我學習幫助很大。有時候就是知道算式怎麼列也知道怎麼解，但數字就是很不好看。不然就是很基礎的運算但是運算量特別大，我覺得這種時候借助科技的力量是比較聰明的，畢竟科技發明出來就是要用嘛。
